pkg/pipeline/extract/conntrack/conntrack.go

```diff
@@ -61,6 +61,11 @@ func (ct *conntrackImpl) Extract(flowLogs []config.GenericMap) []config.GenericM
 
 	var outputRecords []config.GenericMap
 	for _, fl := range flowLogs {
+		if !fl.IsValidProtocol() {
+			log.Debugf("skipping layer2 protocols flow log %v", fl)
+			ct.metrics.inputRecords.WithLabelValues("discarded").Inc()
```

Comment:
```suggestion
			ct.metrics.inputRecords.WithLabelValues("rejected").Inc()
```

Any reason not using the same metric as below ? Else maybe we should make the difference clear for the user

Reply:
I felt rejected because of any error but in this case there is no error just l2 protocol that we aren't supporting but I am up for better suggestion ?

Reply:
What about [a note in the README](https://github.com/netobserv/flowlogs-pipeline/blob/ba6bf34ad04a4ded723579b7a1027d3727f3450c/README.md?plain=1#L650) explaining the diffs then ?

Reply:
Hi @jpinsonneau I checked readme it doesn't really explain any of the metrics ? is there another place where this is already doc ?

Reply:
nope @msherif1234 it's something we should add around the line I mentionned I guess